functions/jest.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('ts-jest').JestConfigWithTsJest} */
+module.exports = {
+    preset: 'ts-jest',
+    testEnvironment: 'node',
+    globalSetup: `<rootDir>/src/tests/setup.ts`,
```

Comment:
Could we use a `beforeAll()` or `beforeEach()` call within the tests instead of having this global setup configured here?  https://jestjs.io/docs/setup-teardown

Somebody reading the test code could easily miss that this setup step exists and have trouble understanding how the tests work.  I think it's much more readable if we can keep the setup closer to the test code.